ncm-network/src/main/perl/network.pm

```diff
@@ -1,31 +1,92 @@
-# ${license-info}
-# ${developer-info}
-# ${author-info}
+#${PMcomponent}
 
-package NCM::Component::network;
+=head1 NAME
 
-#
-# a few standard statements, mandatory for all components
-#
+network: Configure Network Settings
 
-use strict;
-use NCM::Component;
-use vars qw(@ISA $EC);
-@ISA = qw(NCM::Component);
-$EC=LC::Exception::Context->new->will_store_all;
+=head1 DESCRIPTION
+
+The I<network> component sets the network settings through C<< /etc/sysconfig/network >>
+and the various files in C<< /etc/sysconfig/network-scripts >>.
+
+For restarting, a sleep value of 15 is used to make sure the restarted network
+is fully restarted (routing may need some time to come up completely).
+
+Because of this, adding/changing may cause some slowdown.
```

Comment:
yes, of `ncm-ncd`. the network should already be up at this point. i'll rephrase to
```
Because of this additional wait time, adding/changing may cause some delay in the ncmncd run.
```